packages/selectInput/README.md

```diff
@@ -0,0 +1,3 @@
+# SelectInput Name
```

Comment:
@lsamimi - I couldn't find any existing documentation on select inputs. Maybe we should collaborate on some?

Reply:
@mperrotti yes, I'll set up a meeting with you and we can write some doc on this.